doc/tor.md

```diff
@@ -211,3 +209,105 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- In some regions of the world usage of the Tor network is monitored and/or blocked.
```

Comment:
```suggestion
- In some regions of the world the Tor network is monitored and/or blocked.
```

Reply:
Done. Made changes in https://github.com/bitcoin/bitcoin/commit/6bf33db373588582c85ee537cd5e9302c593cb02